src/main/java/codeu/model/data/User.java

```diff
@@ -106,4 +112,34 @@ public String getTime() {
     return date;
   }
 
+  /** When a User is initialized, this method creates a private Conversation with every other User that already exists. */
+  public void createConversations() {
+    // get all USers from UserStore
+    // for each User, create a new Conversation with this User and set to private
+    // add all Conversations to ConversationStore
+    // writethrough to DataStore
+    UserStore userStore = UserStore.getInstance();
+    List<User> users = userStore.getUsers();
+    ConversationStore conversationStore = ConversationStore.getInstance();
+    List<Conversation> conversations = conversationStore.getAllConversations();
+
+    for (User u: users) {
+      String otherUserName = u.getName();
+      String firstUser;
+      String secondUser;
+      /** Gets the conversation link by the names of the two users in alphabetical order */
+       if ((this.name).compareTo(otherUserName) < 0) {
```

Comment:
Please adjust the indentation here. Inside the brackets should be indented 2 more spaces

Reply:
fixed!